packages/strapi-parts/src/Avatar/Avatar.stories.mdx

```diff
@@ -12,7 +13,17 @@ This is the doc of the `Avatar` component
 
 ## Avatar
 
-Description...
+Avatars are used to show a thumbnail of an asset or an individual. 
+
+**Best practices**
+
+- Use the thumbnail preview feature only when the avatar might not be enough.
+- Use avatar in small areas such as tables.
+- In a form we should rather use the Card component.
```

Comment:
I'm not sure this is a valid call, a form is semantic. And a card has nothing to do with a form 😬

Reply:
I'm not sure to understand the final point 😛 . Does it mean that a Card should be inside a form, like a form input of some kind?

Reply:
Actually I think I made a mistake between Card and Carousel components 😅